test/ably/restinit_test.py

```diff
@@ -14,36 +13,77 @@
 
 class TestRestInit(unittest.TestCase):
     def test_key_only(self):
-        AblyRest(Options.with_key(test_vars["keys"][0]["key_str"]))
+        ably = AblyRest(key=test_vars["keys"][0]["key_str"])
+        self.assertEqual(ably.options.key_name, test_vars["keys"][0]["key_name"],
+                         "Key id does not match")
+        self.assertEqual(ably.options.key_secret, test_vars["keys"][0]["key_secret"],
+                         "Key value does not match")
```

Comment:
I will merge, and not that important really, but the test text still refers to key id and key value